### PR TITLE
#161587491 Refactor tests

### PR DESCRIPTION
**Pull request by Georgeygigz**
**What does PR do?**
Refactors all tests

**Completed tasks**
1. Refactoring tests to make them pass
2. Remove unused methods and 

** PT stories**
#161587491

### CODE REVIEW

File: app/api/v2/models/store_model.py [hunk 1]
```diff
@@ -56,6 +56,19 @@ def update_product(self, product_id, product_name, category_id, stock_amount, pr
...
+        database = self.db
+        try:
+            curr = database.cursor()
+            query = "UPDATE products SET stock_amount=%s WHERE product_name=%s;"
+            curr.execute(query, (stock_amount,product_name))
+            database.commit()
+            curr.close()
+            return {"Message": "Product Updated successfully"}, 201
+        except Exception as e:
+            print(e)
```
Comment (WinstonKamau) [inline app/api/v2/models/store_model.py:71]: Exceptions should be returned rather than printed

File: app/api/v2/utils/authorization.py [hunk 2]
```diff
@@ -12,6 +12,7 @@ def admin_required(func):
     @wraps(func)
     def wrapper_function(*args, **kwargs):
         cur_user=[user for user in users if user['email']==get_jwt_identity()]
+        print(cur_user)
```
Comment (WinstonKamau) [inline app/api/v2/utils/authorization.py:15]: remove print statements on the application

File: app/api/v2/views/store_views.py [hunk 3]
```diff
@@ -82,49 +87,56 @@ def put(self,product_id):
...
     @admin_required
     def post(self):
         """ Make a new sale record."""
-        sales_record=Sales().get_all_sales()
+        sales_record = Sales().get_all_sales()
         products = Products().get_all_products()
         current_date = str(date.today())
         data = request.get_json(force=True)
-        #Validate().validate_empty_sales_inputs(data)
+        # Validate().validate_empty_sales_inputs(data)
```
Comment (WinstonKamau) [inline app/api/v2/views/store_views.py:134]: remove commented out methods that are not being used `#` this should be for comments and not functions you are not using.

File: app/tests/v2/base_test.py [hunk 4]
```diff
@@ -51,95 +60,63 @@ def setUp(self):
...
+        result = json.loads(response.data.decode('utf-8'))
+        return result
+
+
     def get_user_token(self):
         '''Generate Token'''
         resp_login = self.user_login()
         token = resp_login.get("token")
-
+        print(token)
```
Comment (WinstonKamau) [inline app/tests/v2/base_test.py:88]: remove print statements

File: app/api/v2/models/store_model.py (hunk 1 above)
Comment (Georgeygigz) [reply app/api/v2/models/store_model.py:71]: Okay thanks I will work on it

**Pull request merged**